core-services/prow/02_config/_config.yaml

```diff
@@ -1115,6 +1110,22 @@ tide:
     repos:
     - redhat-operator-ecosystem/playground
     - redhat-operator-ecosystem/release
+  - labels:
+    - lgtm
+    - approved
+    - bugzilla/valid-bug
```

Comment:
Needs to be `bugzilla/invalid-bug` in `missingLabels` actually - you do not want to block on *not* having a valid bug.

Reply:
ah I got it right at first and thoughts differently, thanks!